src/lib.rs

```diff
@@ -11,10 +11,9 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![cfg_attr(feature = "alloc", feature(alloc))]
 #![cfg_attr(feature = "nightly", feature(i128_type))]
+#![cfg_attr(feature = "nightly", feature(cfg_target_feature))]
 #![cfg_attr(feature = "bench", feature(test))]
-#![cfg_attr(all(feature = "nightly", feature = "std"), feature(zero_one))]
```

Comment:
We got rid of `zero_one`? (What even was using it?)

Reply:
I think we used it for `FieldElement`s a year ago